electron/main.js

```diff
@@ -128,6 +128,9 @@ ipcMain.on('google-auth-request', function(event) {
 ///////////////////////////////////////////////////////////////////////////////
 // APP Windows
 ///////////////////////////////////////////////////////////////////////////////
+
+const iconExt = (process.platform === 'win32') ? 'ico' : 'png';
```

Comment:
on a second thought we could make it better :) add that on line 44 and use the `ICON_PATH` instead..

```js
const ICON = 'wire.' + ((process.platform === 'win32') ? 'ico' : 'png');
const ICON_PATH = path.join(APP_PATH, 'img', ICON);
```